src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { styled } from '@linaria/react'
+import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../styles/media'
+import { ElButtonGroup } from '../button-group'
+import { css } from '@linaria/core'
+
+export const ElTopBarProfile = styled.div`
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px);
+`
+
+export const ElTopBarMainNav = styled(ElButtonGroup)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: var(--spacing-6, 24px);
+  flex-grow: 1;
+`
+
+export const ElTopBarSecondaryNav = styled(ElButtonGroup)`
+  flex-wrap: nowrap;
+  padding-right: var(--spacing-spacing-2, 8px);
+`
+
+export const ElTopBar = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: var(--size-10, 56px);
+  padding: var(--spacing-2, 8px) var(--spacing-5, 20px);
+  border-bottom: var(--border-default, 1px) solid var(--outline-default, #e5e9ed);
+  background: var(--fill-white, #fff);
+
+  ${isDesktopOrBelow} {
+    padding: var(--spacing-2, 8px) var(--spacing-4, 16px);
+
+    ${ElTopBarMainNav}, ${ElTopBarSecondaryNav} {
+      display: none;
+    }
+  }
+
+  ${isTabletOrBelow} {
+    ${ElTopBarProfile} {
+      display: none;
+    }
+  }
+`
+
+export const ElTopBarSearch = styled.div`
+  padding-right: var(--spacing-spacing-2, 8px);
```

Comment:
@nurm717123 Here, too.

**suggestion:** Please review all variables implemented in this ticket and cross-reference with the token file.